src/app/cli/src/coda_main.ml

```diff
@@ -612,7 +606,23 @@ struct
     module Diff_mutant = Diff_mutant
 
     let max_length = max_length
-  end)
+  end
+
+  module Transition_frontier =
+    Transition_frontier.Make (Transition_frontier_inputs)
+
+  module Transition_frontier_persistence = struct
+    module Worker = Transition_frontier_persistence_rpc_worker.Make (struct
```

Comment:
This seems to have the same inputs as `Transition_frontier_persistence`, are you sure this functor is necessary?